lib/mayaUsd/render/mayaToHydra/renderOverride.cpp

```diff
@@ -583,7 +572,46 @@ MStatus MtohRenderOverride::Render(const MHWRender::MDrawContext& drawContext)
 
     params.cullStyle = HdCullStyleBackUnlessDoubleSided;
 
+    int        width = 0;
+    int        height = 0;
+    drawContext.getRenderTargetSize(width, height);
+
+    bool       vpDirty;
+    if ((vpDirty = (width != _viewport[2] || height != _viewport[3]))) {
+        _viewport = GfVec4d(0, 0, width, height);
+        _taskController->SetRenderViewport(_viewport);
+    }
```

Comment:
Yeah, vpDirty is what I want for the camera-prim invalidation.
MFrameContext::kViewMtx would be a transform change already picked up via the dag callback for a camera.

I'll double check the following is correct, but viewport changes dirties projection because of "fill", "overscan", "horizontal", "fitting" and how it can relate to the **fov** not of the camera, but inorder to to match the OpenGL viewport of the camera.  i.e. nothing has changed parameter wise, but we still want the render delegate to pull a new value for the **projectionMatrix** or **focalLength** to match OpenGL.